test/jdk/sun/management/jdp/JdpDefaultsTest.java

```diff
@@ -30,7 +30,7 @@
 /*
  * @test JdpDefaultsTest
  * @summary Assert that we can read JDP packets from a multicast socket connection, on default IP and port.
- *
+ * @requires os.version != "Mac OS X 15.0.1 (aarch64)"
```

Comment:
Unless there is a known reason why this test cannot run on this platform, this is probably not the appropriate fix here. I'd suggest keeping the CR open and problem listing for macosx-aarch64.

Reply:
Ok, I was thinking about debating that after many failures on 15.0.1 and nothing else... But just now we see one on "Mac OS X 12.7.1 (aarch64)" so problemlisting for macosx-aarch64 generally now seems right! (will update)